### ISSUE TITLE
Terminal unresponsive when using Crossterm and Cursive crossterm-backend

### ISSUE THREAD

**Issue by joelparkerhenderson** [crossterm-rs/crossterm#725]
**Describe the bug**

Failure when using Crossterm and Cursive, when enabling Cursive feature `crossterm-backend`. 

Specifically, Cursive launches the TUI, but it doesn't respond to key presses. For comparison, the same app works fine with other Cursive backends, including ncurses, pancurses, termion.

**To Reproduce**

Minimal reproduction code.

File `Cargo.toml` includes:

```toml
[dependencies]
cursive = { version = "*", features = ["crossterm-backend"] }
```

File `main.rs`:
```
fn main() {
    let mut c = cursive::crossterm();
    c.add_global_callback(cursive::event::Key::Esc, |c| c.quit());
    c.run();
}
```

**Expected behavior**

Expected: When I press 'q' to quit, then Cursive quits. 

Actual: When I press 'q' to quit, nothing happens.

I've confirmed this problem happens with other kinds of UI/UX, such as other keypresses, and also by using other Cursive functionality, such as a TextView, or EditView, or Button, etc.

**OS**

macOS Ventura 13.0 on a MacBook Pro with Apple M1 Max chip

**Terminal/Console**

macOS Terminal app i.e. the default macOS terminal application

**Question**

When the Crossterm documentation states "It supports all UNIX and Windows terminals down to Windows 7", does Crossterm currently support UNIX on macOS with Apple M1? I ask because I see some past posts that discuss Crossterm macOS M1 issues/incompatibilities/roadmaps.

If yes, then what's the next step please, to troubleshoot this bug?

If no, then could a maintainer edit to the README and documentation to note that Crossterm is currently not intending to be run on macOS with Apple M1? 

Thank you for your consideration.

**Comment by sigmaSd** [comment]
I doubt that the issue lies in crossterm, you can make sure by testing the examples provided in this repo.

**Comment by joelparkerhenderson** [comment]
Thank you. Same problem happens in the Crossterm examples.

Reproduce:

```
cd examples/interactive-demo
cargo build
```

This succeeds:

```
./target/debug/interactive-demo
```

This hangs:

```
echo "" | ./target/debug/interactive-demo
```

**Comment by sigmaSd** [comment]
Yeah that's a known issue on Mac os, there is a PR that should improve things https://github.com/crossterm-rs/crossterm/pull/711

**Comment by joelparkerhenderson** [comment]
Thank you for the fast info-- that is very helpful to know. Feel free to close this issue because it's already a known issue.
